src/shared/components/sign/sign-view.html

```diff
@@ -50,7 +50,7 @@
     	
         <p ng-if="!ctrl.loaded" flex>Loading schools...</p>
 
-		<md-input-container flex layout="column" ng-if="ctrl.loaded &amp;&amp; currentUser.yearOfBirth >= 1996 &amp;&amp; currentUser.yearOfBirth <= 2004">
+		<md-input-container flex layout="column" ng-if="currentYear - currentUser.yearOfBirth <= 20 &amp;&amp; currentYear - currentUser.yearOfBirth  >= 10">
```

Comment:
@blueset @SingaporeClouds 

It's now requiring people born in years between 1995 and 2005 to pick a school. Is it ok?